### DISCUSSION TITLE
Backtesting Timeframe

### DISCUSSION THREAD

**Discussion by naujoks-stefan** [markusaksli/TradeBot#47] [General]
Thank you very much for the great project. I have a question about backtesting. If I read source code correctly, then you always take 5min timeframe. Did I understand that correctly?

Best
Stefan

**Comment by markusaksli** [comment]
Yes, the current version does not support selecting a timeframe

**Comment by rogegla** [comment]
How can I adjust the timeframe?

**Reply by sanjay762k** [reply]
You need to change in the code the interval from FIVE_MINUTES to other time. Search for FIVE_MINUTES in the code and you will know more.
